tutorials/uniformSampling/tutorial_uniformSampling.m

```diff
@@ -119,8 +124,13 @@
 % Flux variability analysis (FVA) returns the minimum and maximum possible flux 
 % through every reaction in a model.
 
-[minUn, maxUn] = fastFVA(unlimitedOx, 100);
-[minLim, maxLim] = fastFVA(limitedOx, 100);
+if options.useFastFVA
+    [minUn, maxUn] = fastFVA(unlimitedOx, 100);
+    [minLim, maxLim] = fastFVA(limitedOx, 100);
+else
+    [minUn, maxUn] = fluxVariability(unlimitedOx);
```

Comment:
I would prefer to also pass on the option `useFastFVA` explicitly to `fluxVariability()`. @syarra?